src/artifacts/build_dynamic_archive.ts

```diff
@@ -1,13 +0,0 @@
-// deno task artifacts:build:runtime_archive
-//
-// Generates a JSON file with all of the files that need to be accessed dynamically.
-
-import { resolve } from "../deps.ts";
-import { buildArtifacts, rootSrcPath } from "./util.ts";
-
-await buildArtifacts({
-	rootPath: resolve(rootSrcPath(), "src", "dynamic"),
-	patterns: ["*.ts"],
-	outputPath: resolve(rootSrcPath(), "artifacts", "dynamic_archive.json"),
-	encode: "string",
-});
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 1 to 14]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->See above comment<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/rivet-gg/opengb/417?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->